beacon.lua

```diff
@@ -18,7 +22,8 @@ for _,protected in pairs({true, false}) do
 	})
 
 	minetest.register_node("telemosaic:beacon"..node_name_suffix, {
-		description = description_prefix.."Telemosaic Beacon Active (you hacker you!)",
+		description = S(description_prefix ..
+			"Telemosaic Beacon Active (you hacker you!)"),
```

Comment:
Probably could remove the `(you hacker you!)` part, to make translations simpler.

Reply:
It's some kind of a tradition or pattern I think - Advtrains has this for the mesecon detector track. That makes me think there may be others. But it's overall not *that* common I think. I'm not sure how well it translates, whether to an outsider it seems more native to English or rather more universal.

Reply:
Doing some research, it's widely used in (pretty much?) all of cheapie's mods - mesecons, digilines, digistuff and others, and also in pipeworks by other authors DS and Novatux. It seems like a tradition work keeping..

Reply:
I asked about on IRC, apparently the engine had it as in the description for air until 2018: https://irc.minetest.net/minetest/2024-02-04#i_6150100

Reply:
Interesting, I guess we'll keep it then :)